apps/argus/src/keeper/keeper_metrics.rs

```diff
@@ -11,273 +10,169 @@ use {
 };
 
 #[derive(Clone, Debug, Hash, PartialEq, Eq, EncodeLabelSet)]
-pub struct AccountLabel {
+pub struct ChainIdLabel {
     pub chain_id: String,
-    pub address: String,
 }
 
 #[derive(Clone, Debug, Hash, PartialEq, Eq, EncodeLabelSet)]
-pub struct ChainIdLabel {
+pub struct SubscriptionIdLabel {
+    pub chain_id: String,
+    pub subscription_id: String,
+}
+
+#[derive(Clone, Debug, Hash, PartialEq, Eq, EncodeLabelSet)]
+pub struct PriceFeedIdLabel {
     pub chain_id: String,
+    pub price_feed_id: String,
+}
+
+#[derive(Clone, Debug, Hash, PartialEq, Eq, EncodeLabelSet)]
+pub struct KeeperIdLabel {
+    pub chain_id: String,
+    pub keeper_id: String,
 }
 
 pub struct KeeperMetrics {
-    // TODO: reevaluate what metrics are useful for argus
-    pub current_sequence_number: Family<AccountLabel, Gauge>,
-    pub end_sequence_number: Family<AccountLabel, Gauge>,
-    pub balance: Family<AccountLabel, Gauge<f64, AtomicU64>>,
-    pub collected_fee: Family<AccountLabel, Gauge<f64, AtomicU64>>,
-    pub current_fee: Family<AccountLabel, Gauge<f64, AtomicU64>>,
-    pub target_provider_fee: Family<AccountLabel, Gauge<f64, AtomicU64>>,
-    pub total_gas_spent: Family<AccountLabel, Gauge<f64, AtomicU64>>,
-    pub total_gas_fee_spent: Family<AccountLabel, Gauge<f64, AtomicU64>>,
-    pub requests: Family<AccountLabel, Counter>,
-    pub requests_processed: Family<AccountLabel, Counter>,
-    pub requests_processed_success: Family<AccountLabel, Counter>,
-    pub requests_processed_failure: Family<AccountLabel, Counter>,
-    pub requests_reprocessed: Family<AccountLabel, Counter>,
-    pub reveals: Family<AccountLabel, Counter>,
-    pub request_duration_ms: Family<AccountLabel, Histogram>,
-    pub retry_count: Family<AccountLabel, Histogram>,
-    pub final_gas_multiplier: Family<AccountLabel, Histogram>,
-    pub final_fee_multiplier: Family<AccountLabel, Histogram>,
-    pub gas_price_estimate: Family<AccountLabel, Gauge<f64, AtomicU64>>,
-    pub accrued_pyth_fees: Family<ChainIdLabel, Gauge<f64, AtomicU64>>,
+    /// Number of active subscriptions per chain
```

Comment:
for the future i recommend you look at price pusher metrics and get some inspirations.